### DISCUSSION TITLE
Polarfire Soc Thermal Specifications

### DISCUSSION THREAD

**Discussion by carrd7** [polarfire-soc/polarfire-soc-documentation#316] [General] [answered]
What are the temperature limits for the Microchip PolarFire SoC FPGA? I would like to know the temperatures that it can hold for the operational and non-operational status.

I found this in a datasheet and I would like to confirm if it's correct:
Operational: -40°C to +100°C.
Non-operational: -65°C to +150°C.

**Comment by hughbreslin** [comment, marked as answer]
Hey @carrd7 

It depends on which device you choose, for -40°C to +100°C you'll need an industrial grade part:

| Device Options | Extended Commercial Temperature (E) 0 °C–100   °C | Industrial Temperature (I) –40 °C–100 °C |
|----------------|---------------------------------------------------|------------------------------------------|
| MPFS250T       | Yes                                               | Yes                                      |
| MPFS250TL      | Yes                                               | Yes                                      |
| MPFS250TS      | No                                                | Yes                                      |
| MPFS250TLS     | No                                                | Yes                                      |

Full info in section 9 of this [document](https://ww1.microchip.com/downloads/aemDocuments/documents/FPGA/ProductDocuments/DataSheets/60001656E.pdf)
